concordia/templates/transcriptions/campaign_detail.html

```diff
@@ -14,8 +14,22 @@
     <div class="row">
         <div class="col-12">
             <h1 class="my-default ml-3">{{ campaign.title }}</h1>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col-9">
             <p class="ml-default mb-default hero-text">{{ campaign.description }}</p>
         </div>
+        <div class="col-3 related-links">
+            <h4>Related Links</h4>
+            <ul>
+            {% for resource in campaign.resource_set.all %}
+            <li>
+            <a href="{{ resource.resource_url }}" target="_blank">{{ resource.title }} &raquo;</a>
```

Comment:
Since we're making the links open in a new window, should we add a [`external-link`](https://fontawesome.com/icons/external-link-alt?style=solid) or [`clone`](https://fontawesome.com/icons/clone?style=regular) icon?